crates/re_types_builder/src/codegen/docs/mod.rs

```diff
@@ -0,0 +1,170 @@
+use crate::codegen::common::ExampleInfo;
+use crate::objects::FieldKind;
+use crate::CodeGenerator;
+use crate::Object;
+use crate::ObjectField;
+use crate::ObjectKind;
+use crate::Objects;
+use crate::Reporter;
+use camino::Utf8PathBuf;
+use std::collections::BTreeSet;
+use std::fmt::Write;
+
+use super::common::get_documentation;
+
+macro_rules! putln {
+    ($o:ident) => {let _ = writeln!($o);};
+    ($o:ident, $($tt:tt)*) => {let _ = writeln!($o, $($tt)*);};
```

Comment:
nit, see `CODE_STYLE.md`

```suggestion
    ($o:ident) => { writeln!($o).ok(); };
    ($o:ident, $($tt:tt)*) => { writeln!($o, $($tt)*).ok(); };
```

Reply:
also, just because you _can_ omit whitespace doesn't mean you _should_ :)